### DISCUSSION TITLE
can we filter on Realtime subscriptions?

### DISCUSSION THREAD

**Discussion by oSethoum** [pocketbase/pocketbase#521] [Q&A] [answered]
Is it possible to have a filter on what you want to subscribe to? 
example:
```javascript
client.realtime.subscribe('resource', {
filter: 'created >= "2022-01-01 00:00:00"',
} function (e) {
    console.log(e.record);
});
```

**Comment by ganigeorgiev** [comment, marked as answer]
No. We may add it in the future, but currently this is not supported. Realtime events can be filtered only via the API rules.

For more info you could check  https://github.com/pocketbase/pocketbase/discussions/304.
